packages/react-dom-bindings/src/client/ReactDOMTextarea.js

```diff
@@ -120,8 +95,10 @@ export function initWrapperState(element: Element, props: Object) {
     initialValue = defaultValue;
   }
 
+  const stringValue = getToStringValue(initialValue);
+  node.defaultValue = (stringValue: any); // This will be toString:ed.
```

Comment:
This used to be set by setting the children in getHostProps which used `textContent` which is the same as setting `defaultValue` of a text area.